spec/features/articles_list_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'ffaker'
+
+describe "Listing articles" do
+  let!(:article) { create :article }
+  scenario "someone views the articles index" do
+    visit articles_path
+    expect(page).to have_content "Articles List"
+  end
+
+  scenario "see existing articles" do
+    visit articles_path
+    expect(page).to have_content article.title
+    expect(page).to have_content article.text.split(' ').first(100).join(' ')+'...'
+    save_and_open_page
+  end
+
+  scenario "view full article" do
+    visit articles_path
+    expect(page).to have_link "more"
```

Comment:
This could be written as `expect(page).to have_link "more", href: article_path article` to be more specific. It could also be moved into the index view test above, then in this one you would just start off by visiting the article_path for the full article.